contracts/guess-the-number/src/lib.rs

```diff
@@ -32,4 +25,34 @@ impl GuessTheNumber {
     }
 }
 
+pub const ADMIN_KEY: &Symbol = &symbol_short!("ADMIN");
+
+pub trait Administratable {
+    fn get_admin(env: &Env) -> Option<Address> {
+        env.storage().instance().get(ADMIN_KEY)
+    }
```

Comment:
This does not match the existing [admin-sep implementation](https://github.com/theahaco/admin-sep/blob/main/admin_sep/src/administratable.rs), which uses just `admin` as the getter. My understanding is that Leigh requested this specifically, though I'm not sure where.